server/resp/pom.xml

```diff
@@ -3,6 +3,7 @@
          xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
    <modelVersion>4.0.0</modelVersion>
    <parent>
+      <?m2e execute onConfiguration?>
```

Comment:
Should revert?

Reply:
ops...again..., thanks! I try hard but it always find a way to fool me :)